CMakeLists.txt

```diff
@@ -120,6 +94,34 @@ if(VTK_VERSION VERSION_LESS 9.3.20240914)
   find_package(VTK REQUIRED COMPONENTS opengl)
 endif()
 
+# Modules
+option(F3D_MODULE_RAYTRACING "Raytracing module" OFF)
+option(F3D_MODULE_EXR "OpenEXR images module" OFF)
+# Needs vtkResourceStream (https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9663)
+cmake_dependent_option(F3D_MODULE_WEBP "WebP images module" OFF "VTK_VERSION VERSION_GREATER_EQUAL 9.2.20221216" OFF)
```

Comment:
I wonder if it would be better to check and fail if enabled instead of silentely ignoring the option